frontend/pkg/frontend/middleware_logging.go

```diff
@@ -86,6 +87,19 @@ func MiddlewareLoggingPostMux(w http.ResponseWriter, r *http.Request, next http.
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(attribute.String("correlation.id", correlationData.CorrelationRequestID))
 	span.SetAttributes(attribute.String("client.request.id", correlationData.ClientRequestID))
+	cID, err := baggage.NewMember("correlation.id", correlationData.CorrelationRequestID)
+	if err != nil {
+		span.AddEvent("unable to create correlationID baggage member")
+	}
+	rID, err := baggage.NewMember("request.id", correlationData.RequestID.String())
```

Comment:
in relation to  https://github.com/Azure/ARO-HCP/pull/1185, I wonder if we need to store the request ID in the baggage since CS can read it from the X-Request-ID header. But adding `ClientRequestID` would be useful.

Reply:
That probably works too. The idea here was to transfer all request-specific data points for the correlation in the same way.

Reply:
sounds good to me.